### DISCUSSION TITLE
ALVR Support on Mac though SteamVR macOS beta

### DISCUSSION THREAD

**Discussion by asingueb** [alvr-org/ALVR#1169] [General]
For those who didn't know, there's an option in "Properties" on SteamVR that lets you use beta versions of SteamVR, and some of those betas are compatible with Mac. An ALVR program on Mac would be helpful to use different types of VR Headsets with SteamVR on Mac.

**Comment by zmerp** [comment]
That version of SteamVR is too old for ALVR needs. And frame grabbing will likely not work and this is also the most difficult part that we had to face for Linux support (we need to inject code into SteamVR compositor using a Vulkan layer and then grab the Vulkan Image and pose matrix using libunwind). There is no chance macos can have the same amount of development time.

**Comment by WiBlum** [comment]
Maybe via Wine using KegWorks, Porting Toolkit, Whisky, Proton or Virtualisation&Emulation this would work, I honestly prefer having a native working app or a working app through wine instead of running it though windows ....
I'm actually trying to make it work on those Translation app to have steamvr working with alvr on wine and launch the game, but I honestly have to clue if it's gonna work or break the whole thing, I've to test at least 5 different methods and see if it needs a patch or some king of custom config on winetricks.

See ya later
